pkg/controllers/osc/resources/operating_system_config.go

```diff
@@ -80,14 +78,9 @@ func OperatingSystemConfigCreator(
 				return nil, fmt.Errorf("failed to decode provider configs: %v", err)
 			}
 
-			var cloudConfig string
-			if providerConfig.OverwriteCloudConfig != nil {
-				cloudConfig = *providerConfig.OverwriteCloudConfig
-			} else {
-				cloudConfig, err = cloudprovider.GetCloudConfig(providerConfig, md.Spec.Template.Spec.Versions.Kubelet)
-				if err != nil {
-					return nil, fmt.Errorf("failed to fetch cloud-config: %v", err)
-				}
+			cloudConfig, err := cloudprovider.GetCloudConfig(providerConfig, md.Spec.Template.Spec.Versions.Kubelet)
+			if err != nil {
+				return nil, fmt.Errorf("failed to fetch cloud-config: %v", err)
```

Comment:
Can you please revert back to the previous changes here?

Reply:
Hmm strange, I am pretty much sure I got rid of it. Looks like goland git change list isn't working as it should 🤔

Reply:
Done